packages/language-server/package.json

```diff
@@ -25,7 +25,7 @@
     "ts-morph": "^12.0.0",
     "typescript": "^4.5.4",
     "vscode-css-languageservice": "^5.1.1",
-    "vscode-emmet-helper": "2.1.2",
+    "@vscode/emmet-helper": "^2.8.4",
```

Comment:
Nice, do these add new features?

Reply:
They don't have a changelog but as far as I could tell from the commit history, it was only fixes